src/components/DataMessenger/DataMessenger.js

```diff
@@ -207,6 +210,26 @@ export default class DataMessenger extends React.Component {
       .catch((error) => {
         console.error(error)
       })
+
+    if (this.props.enableQueryQuickStartTopics) {
+      fetchTopics(getAuthentication(this.props.authentication))
+        .then((response) => {
+          this.setState({
+            topics: _get(response, 'data.data.topics'),
+          })
+          if (_get(this.state, 'topics.length') > 0) {
```

Comment:
If you are checking for a state variable immediately after setting the state, it doesn't always behave like you think. It is better to set the initial topics state inside the constructor